subprojects/docs/src/docs/userguide/jvm/toolchains.adoc

```diff
@@ -15,31 +15,36 @@
 [[toolchains]]
 = Toolchains for JVM projects
 
-By default, Gradle uses the same Java version for running Gradle itself and building JVM projects.
+Working on multiple projects can require interacting with multiple versions of the Java language.
+Even within a single project different parts of the codebase may be fixed to a certain language level due to backward compatibility requirements.
+Because of this, different versions of the same tools (a toolchain) must be installed and managed on each machine that builds the project.
 
-This is not always desirable.
+A **Java toolchain** is a set of tools to build and run Java projects, which is usually provided by the environment via local JRE or JDK installations.
+Compile tasks may use `javac` as their compiler, test and exec tasks may use the `java` command while `javadoc` will be used to generate documentation.
+
+By default, Gradle uses the same Java toolchain for running Gradle itself and building JVM projects.
+However, this may not be always desirable.
 Building projects with different Java versions on different developer machines and CI servers may lead to unexpected issues.
 Additionally, you may want to build a project using a Java version that is not supported for running Gradle.
 
-A Java Toolchain (from now on referred to simply as toolchain) is a set of tools, usually taken from a local JRE/JDK installation that are used to configure different aspects of a build.
-Compile tasks may use `javac` as their compiler, test and exec tasks may use the `java` command while `javadoc` will be used to generate documentation.
+In order to improve reproducibility of the builds and make build requirements clearer, Gradle allows to configure toolchains on both project and task level.
 
 [[sec:consuming]]
-== Consuming Toolchains
+== Toolchains for projects
 
-A build can globally define what toolchain it targets by stating the Java Language version it needs and optionally the vendor:
+A build can globally define what toolchain it targets by stating the Java language version it needs:
```

Comment:
You added project to the title, to clarify scope. I would then remove "build" from here as we are discussing project level configuration.

And we might want one day to introduce a toolchain block in settings to have real build level configuration.